src/TelemetryChannels/ServerTelemetryChannel/Shared/Implementation/TelemetryChannelEventSource.cs

```diff
@@ -30,6 +30,26 @@ public bool IsVerboseEnabled
             }
         }
 
+        // Verbosity is Error - so it is always sent to portal; Keyword is Diagnostics so throttling is not applied.
+        [Event(1,
+            Message = "It is a diagnostics message. Backoff logic was disabled. Transmission will be resolved.",
```

Comment:
It means that we were not sending data and now we disable backoff logic and resolving sending data on the regular interval.

---
In reply to: [69644269](https://github.com/Microsoft/ApplicationInsights-dotnet/pull/248#discussion_r69644269) [](ancestors = 69644269)